cf_cli_java_plugin.go

```diff
@@ -168,10 +177,23 @@ func (c *JavaPlugin) execute(commandExecutor cmd.CommandExecutor, uuidGenerator
 	}
 
 	var remoteCommandTokens = []string{JavaDetectionCommand}
-
+	remoteFileFullPath := ""
+	localFileFullPath:= ""
 	switch command {
 	case heapDumpCommand:
-		heapdumpFileName := "/tmp/heapdump-" + uuidGenerator.Generate() + ".hprof"
+
+		valid, _ :=utils.CheckRequiredTools(applicationName)
+		if !valid{
+			return "", nil
+		}
+		
+		fspath, _ := utils.GetAvailablePath(applicationName, remoteDir)
+		//fmt.Println(fspath)
```

Comment:
Please remove this line or use a logger with log level "DEBUG" instead.

Reply:
removed